packages/components/package.json

```diff
@@ -77,14 +77,15 @@
   "scripts": {
     "start": "cross-env REACT_APP_FAST_REFRESH=false && start-storybook -p 6006 -s public",
     "build:cjs": "rm -rf dist/src && rm -f dist/tsconfig.tsbuildinfo && tsc -b",
+    "build:esm": "tsc --project ./tsconfig.esm.json",
     "build:css": "postcss ./src/styles/main.css -o ./dist/main.css",
     "build:storybook": "build-storybook -s public",
-    "build": "yarn run build:cjs && yarn run build:css && yarn run build:storybook",
+    "build": "yarn run build:cjs && yarn build:esm && yarn run build:css && yarn run build:storybook",
     "bundle": "webpack",
     "all": "yarn bundle && yarn build",
     "lint": "prettier --check . && eslint --ignore-path .gitignore .",
     "format": "prettier --write .",
-    "prepack": "yarn run build:cjs && yarn run bundle",
+    "prepack": "yarn run build:cjs && yarn build:esm",
```

Comment:
This would remove the webpack bundle from NPM, I believe. I agree it should be removed, but are you sure it won't break your Observable setup?